dev-commands/reset-db-import-sql.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+DC="$(dirname $0)/../dc-dev.sh"
+FILE="$1"
+
+[ -n "$FILE" -a -r "$FILE" ] || {
+  echo "ERROR: cannot read file $FILE"
+  exit 1
+}
+
+$DC rm -svf postgres
+$DC up -d postgres
```

Comment:
you can also call the `truncate_db` route of the writer, which is only enabled in dev mode. See https://github.com/OpenSlides/openslides-datastore-service/blob/main/docs/development.md#development-exclusive-tools

Reply:
I tried this and it generally seems to work.
I think I still prefer my way of completely resetting postgres here.
This script is meant to import complete DB-dumps and only using the truncate_db route could result in many errors.
Though there were many "relation already exists" messages from CREATE statements,
I also saw some constraint violations, which don't happen when postgres is completely fresh. Also it's only a time loss of a few seconds.

Reply:
ok